spec/requests/api/v0.1/post_hesa_trainees_spec.rb

```diff
@@ -68,6 +73,12 @@
       expect(Trainee.last.state).to eq("submitted_for_trn")
     end
 
+    it "sets the correct disabilities" do
+      expect(response.parsed_body["disability_disclosure"]).to eq("disabled")
+      expect(response.parsed_body["disability1"]).to eq("58")
+      expect(response.parsed_body["disability2"]).to eq("57")
```

Comment:
Is there also supposed to be an assertion for the `trainee.disabilities` being set on the record?